os/mm/mm_heap/mm_manage_allocfail.c

```diff
@@ -39,8 +39,14 @@
 #endif
 #endif
 
+#if defined(CONFIG_APP_BINARY_SEPARATION) && defined(__KERNEL__) && defined(CONFIG_DEBUG_MM_HEAPINFO)
+#include "../../kernel/binary_manager/binary_manager.h"
```

Comment:
You can include this file as below:
`#include "binary_manager/binary_manager.h"`